docs/core/testing/selective-unit-tests.md

```diff
@@ -31,7 +31,8 @@ dotnet test --filter <Expression>
   | -------------- | -------------------- |
   | MSTest         | `FullyQualifiedName`<br>`Name`<br>`ClassName`<br>`Priority`<br>`TestCategory` |
   | xUnit          | `FullyQualifiedName`<br>`DisplayName`<br>`Traits` |
-  | Nunit          | `FullyQualifiedName`<br>`Name`<br>`Priority`<br>`TestCategory` |
+  | NUnit          | `FullyQualifiedName`<br>`Name`<br>`Priority`<br>`TestCategory` |
+  | TUnit          | `FullyQualifiedName`<br>`Name`<br>`Category`<br>`Property` |
```

Comment:
Let's take this opportunity to reorder by name the test framework please.

Reply:
Err ignore that 😅 Fixed it.